extensions/amp-story/1.0/amp-story.js

```diff
@@ -448,6 +440,18 @@ export class AmpStory extends AMP.BaseElement {
     if (this.maybeLoadStoryDevTools_()) {
       return;
     }
+
+    const needsDvhPolyfill =
+      !this.win.CSS?.supports?.('height: 1dvh') &&
+      !getStyle(this.win.document.documentElement, '--story-dvh');
+
+    const onResize = (size) => {
+      needsDvhPolyfill && this.polyfillDvh_(size);
+      this.onViewportResize_();
+    };
+
+    this.getViewport().onResize(onResize);
```

Comment:
Does this work? You haven't been binding the context (either fat arrow function or `.bind(this)`)

Reply:
Def worth checking, I don't think arrow functions can have a unique context anyways, so probably it keeps the context from the caller which works in this scenario.

Reply:
Can confirm that the context inside the function `onResize` is the AmpStory instance, so everything works as expected.

Reply:
Thank you for checking 🖤